project.php

```diff
@@ -694,7 +695,14 @@ function do_project_info_table()
                 $blurb = _("Images, Pages Proofread, & Differences");
                 $url2 = "$url&amp;select_by_user";
                 $blurb2 = _("Just my pages");
-                $detail = "<a href='$url'>$blurb</a> &gt;&gt;<a href='$url2'>$blurb2</a>&lt;&lt;";
+                $detail = "<a href='$url'>$blurb</a> | <a href='$url2'>$blurb2</a>";
+                $state_index = array_flip($PROJECT_STATES_IN_ORDER);
+                if($state_index[$project->state] >= $state_index[PROJ_F1_AVAILABLE])
```

Comment:
Using PROJ_F1_AVAILABLE is non-ideal as in theory different sites can customize the different rounds. It would be more round-about, but probably overall better, if you did something like this (untested) within the Project class:
```
    public function has_been_through_formatting_round()
    {
        global $PROJECT_STATES_IN_ORDER;
        $state_index = array_search($this->state, $PROJECT_STATES_IN_ORDER);
        for($i=$state_index; $i>=0; $i--)
        {
            $round = get_Round_for_project_state($PROJECT_STATES_IN_ORDER[$i]);
            if($round)
            {
                if($round->is_formatting_round())
                    return TRUE;
            }
        }
        return FALSE;
    }
```